client/packages/system/src/Item/Components/StockItemSelectModal/StockItemSelectModal.tsx

```diff
@@ -112,6 +112,9 @@ export const StockItemSelectModal = ({
             onChange={onChangeSelectedItems}
             options={options}
             renderOption={renderOption}
+            height={
+              screen.height > 700 ? 365 : 0
```

Comment:
I'm still able to get this by adjusting my chrome window height, so that I can be annoying..

<img width="778" alt="image" src="https://user-images.githubusercontent.com/9192912/212761155-7514bd7f-8c3d-49d7-9681-0476e6bfcdbf.png">

This works for me

```
const MODAL_HEIGHT = 600;
const MODAL_WIDTH = 650;
...

<Modal
      slideAnimation={false}
      title={t('heading.select-items')}
      width={MODAL_WIDTH}
      height={MODAL_HEIGHT}

...

// 285 = the height of components on the select item modal (235px) + the 50px in the useDialog height calc
height={Math.min(window.innerHeight - 285, MODAL_HEIGHT)}
```

Reply:
updated. original comment was incorrect.
The Math.min calculation comes from `useDialog`

Reply:
Looks so much better now~
Thank you 🙏🏻